src/PersonObjects/Sleeve/Sleeve.ts

```diff
@@ -214,23 +214,24 @@ export class Sleeve extends Person implements SleevePerson {
 
     // Set experience/money gains based on class
     let classType: ClassType | undefined;
+    // TODO: why lower case??? It's not effecient, not typesafe and in general a bad idea
     switch (className.toLowerCase()) {
-      case "study computer science":
+      case ClassType.computerScience.toLowerCase():
```

Comment:
You can (and probably should) keep the old case here for backwards compat. Very simple and harms nothing.

Reply:
This fix is a bit relevant to discussions in this ticket: https://github.com/bitburner-official/bitburner-src/issues/372.

Even though it's a game, it still has a publicly documented API which specifies the exact values to provide in scripts.
IMO it would be right to accept only these values and ask players to stick to docs. And maybe we should some time to adjust their scripts by deprecating invalid values and letting them know when there's an issue in a script.